pkgs/applications/networking/instant-messengers/turses/default.nix

```diff
@@ -1,7 +1,9 @@
-{ stdenv, python3Packages }:
+{ stdenv, python36Packages }:
 
 with stdenv.lib;
-with python3Packages;
+
+# Needs Python <3.7 for now, see https://github.com/louipc/turses/issues/4
+with python36Packages;
```

Comment:
`buildPython*` has a `disabled` parameter, where you can condition which python interpreters this will build for.

Reply:
This isn't a python package though, it's just an application using python, so people don't need to select the interpreter

Reply:
Someone can still override it though. This is generally the way we go for anything using `buildPython*` that needs to condition it being built for a certain interpreter version.

Reply:
Ah, so you mean to take a `pythonPackages` argument and override it to `pythonPackages36` in top-level/python-packages.nix?

Reply:
That would be doing `turses = python36Packages.callPackage ../applications/networking...`
but it's less overideable.

But that makes it sound like I mentioned removing this line, we need to keep it, but adding the `disabled` will prevent it from being built on `python37` at all.

```
buildPythonApplication rec {
  disabled = !(python36Packages.pythonOlder "3.7");
}
```

Reply:
So you just want such a `disabled = ...` line?

Reply:
> So you just want such a `disabled = ...` line?

Yep.

Reply:
Did that :)